libs/docs/core/datetime-picker/examples/datetime-format-example/datetime-format-example.component.ts

```diff
@@ -45,15 +46,19 @@ export const CUSTOM_FD_DATETIME_FORMATS: DateTimeFormats = {
         }
     ],
     standalone: true,
-    imports: [DatetimePickerComponent, FormsModule, FdDatetimePipesModule]
+    imports: [DatetimePickerComponent, FormsModule, FdDatetimePipesModule, DatePipe]
 })
 export class DatetimeFormatExampleComponent {
     date = FdDate.getNow();
-    specificDateTimeFormat = {
+    basicDateTimeFormat = {
```

Comment:
Can we add types for year, month, day, hour, and minute? I believe it would be more convenient to have explicit types defined for each property, improving code readability and helping with type safety. @dpavlenishvili

Reply:
The `Intl.DateTimeFormatOptions` interface is part of the ECMAScript Internationalization API, and its properties might change slightly between ECMAScript versions. Anyone can create their own custom interfaces based on `Intl.DateTimeFormatOptions` to include any additional properties they need.